src/main/riddl/Order/Order.riddl

```diff
@@ -4,16 +4,16 @@ context OrderContext is {
   type LineItem is {
     product: SKU,
     quantity: Number,
-    unitPrice: Number,
     lineTotal: Number
   }
 
   type OrderInfo is {
+    order_id: OrderId,
     lineItems: LineItem+,
-    specialInstructions: String,
-    orderTotal: Number
+    specialInstructions: String
```

Comment:
I see. You just moved this attribute. Still, I think special instructions should be optional
specialInstructions: String?

Reply:
@miguelTaningcoYoppworks - has this been done?

Reply:
Updated branch to make this optional ^